plugin/lsp.vim

```diff
@@ -21,6 +21,9 @@ let g:lsp_preview_keep_focus = get(g:, 'lsp_preview_keep_focus', 1)
 let g:lsp_use_event_queue = get(g:, 'lsp_use_event_queue', has('nvim') || has('patch-8.1.0889'))
 let g:lsp_insert_text_enabled= get(g:, 'lsp_insert_text_enabled', 1)
 
+let g:Lsp_get_vim_completion_item = get(g:, 'Lsp_get_vim_completion_item', function('lsp#omni#default_get_vim_completion_item'))
```

Comment:
Can we change this to be an array instead even though we don't support multiple. This is what I do in asyncomplte v2. That way we don't need to worry about the capital L.

```
let g:lsp_get_vim_completion_item = [function('sp#omni#default_get_vim_completion_item')]
```